awesome_owl/static/src/counter/counter.js

```diff
@@ -0,0 +1,16 @@
+import { Component, useState } from "@odoo/owl";
+
+export class Counter extends Component {
+  static template = "awesome_owl.counter";
+
+  static props = {
+    onChange:Function
+  }
+
+  setup() {
+    this.count = useState({ value: 0 });
+  }
+  increment() {
```

Comment:
```suggestion

  increment() {
```
For all the functions